mautrix/api.py

```diff
@@ -20,11 +20,18 @@
 
 from mautrix.errors import make_request_error, MatrixConnectionError, MatrixRequestError
 from mautrix.util.logging import TraceLogger
+from mautrix.util.opt_prometheus import Counter, Histogram
 from mautrix import __version__ as mautrix_version
 
 if TYPE_CHECKING:
     from mautrix.types import JSON
 
+API_CALLS = Counter("bridge_matrix_api_calls",
+                    "The number of Matrix client API calls made", ("method",))
+API_CALLS_FAILED = Counter("bridge_matrix_api_calls_failed",
+                           "The number of Matrix client API calls which failed", ("method",))
+MATRIX_REQUEST_SECONDS = Histogram("bridge_matrix_request_seconds",
```

Comment:
In practise I think we got this wrong.

Reply:
The only incorrect metric is `bridge_matrix_request_seconds`, right?

Reply:
Yeah, the others look fine!

Reply:
I removed `bridge_matrix_request_seconds` from this PR. I may come in another PR.